config/runtimes/kserve-tritonserver.yaml

```diff
@@ -12,10 +12,12 @@ spec:
       version: "2"
     - name: onnx
       version: "1"
+      autoSelect: true
     - name: pytorch
       version: "1"
     - name: triton
       version: "2"
+      autoSelect: true
```

Comment:
Technically triton isn’t a model format, I guess it is here for backwards compatibility?

Reply:
Right, in case someone does `modelFormat.name: triton`.  I'm okay with removing this `triton` entry, but i think `triton` still needs to be in this list to support when the old triton key is specified in the predictor.